### ISSUE TITLE
Why my Airbus_ 330 screen and mcdu panel white

### ISSUE THREAD

**Issue by CA330** [JMaverick16/Airbus_A330#30]
Why my Airbus_ The mcud panel screen of 330 is white and cannot be displayed. Other Du screens are white and unclear

**Comment by legoboyvdlp** [comment]
What version of flightgear?

**Comment by CA330** [comment]
3.5, 3.6, and 3.8 all have this problem

**Comment by SP-NTX** [comment]
@CA330 3.5 from 2013/14, 2018.3.5, 2020.3.5?

**Comment by Octal450** [comment]
Its the display shaders. They just don't work on some planes. On my MDs I added the option to neglect them.

Kind Regards,
Josh

**Comment by CA330** [comment]
Josie, I want to ask how to ignore their options

**Comment by Marsdolphin** [comment]
版本3.5, 3.6, 3.8 是 2018.3.5/2018.3.6, 还是2020.3.5/2020.3.6/2020.3.8?
------------
Do you mean version 2018.3.5/3.6 or 2020.3.5/3.6/3.8?

**Comment by CA330** [comment]
2020.3.8




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年5月13日(星期四) 晚上10:16
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [JMaverick16/Airbus_A330] Why my Airbus_ 330 screen and mcdu panel white (#30)





 
版本3.5, 3.6, 3.8 是 2018.3.5/2018.3.6, 还是2020.3.5/2020.3.6/2020.3.8?
 
您是指版本2018.3.5 / 3.6还是2020.3.5 / 3.6 / 3.8？
 
-
您收到此邮件是因为有人提到您。
直接回复此电子邮件，在GitHub上查看，或取消订阅。

**Comment by Marsdolphin** [comment]
OK
Octal说的是那个白色是屏幕视觉效果，只有在他做的麦道飞机上加了关闭这个效果的选项
你是想问下能不能忽略他们吗？

Translated Octal's message, see original there
Do you want to ask if we can ignore them?

**Comment by Octal450** [comment]
Wym ignore. Set model shader off it'll kill the shaders. I'll add a dedicated option when I revisit it.

Tl:dr you have weird gpu drivers... try without shaders.

Kind Regards,
Josh

**Comment by CA330** [comment]
![捕获](https://user-images.githubusercontent.com/83896034/118347279-8d83f000-b574-11eb-9c04-cbd344be00a1.PNG)
Why do I do this when I turn on ALS

**Comment by CA330** [comment]
Why do I do this when I turn on ALS

**Comment by CA330** [comment]
The top options are black and invisible

**Comment by Marsdolphin** [comment]
你试试在Settings > Additional Settings的框里加这一行
--prop:/sim/gui/current-style=0

Try to pass --prop:/sim/gui/current-style=0 to FG

**Comment by CA330** [comment]
Thank you. My problem has been solved, but can you tell me why my airbus-330 painting page doesn't pop up?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Josh ***@***.***&gt;; 
发送时间: 2021年5月14日(星期五) 中午12:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [JMaverick16/Airbus_A330] Why my Airbus_ 330 screen and mcdu panel white (#30)





 
Wym ignore. Set model shader off it'll kill the shaders. I'll add a dedicated option when I revisit it.
 
Tl:dr you have weird gpu drivers... try without shaders.
 
Kind Regards,
 Josh
